elog/pswww.py

```diff
@@ -154,9 +156,9 @@ def get_experiment_logbook(self, instrument, station=None):
 
         logger.debug("Requesting current experiment for %s", instrument)
         # Format correct URL
-        url = '{url}/lgbk/lgbk/ws/activeexperiment_for_instrument_station' \
+        url = '{url}/{lgbk}/lgbk/ws/activeexperiment_for_instrument_station' \
               '?instrument_name={instrument}' \
-              ''.format(url=self._url, instrument=instrument)
+              ''.format(url=self._url, lgbk=self._lgbk, instrument=instrument)
```

Comment:
I know this isn't your original code, but while you're editing this, how about this more-readable version:
```python
        url = (
            f"{self._url}/{self._lgbk}/lgbk/ws/"
            f"activeexperiment_for_instrument_station"
            f"?instrument_name={instrument}"
         )
```